src/screens/BlaireHome/BlaireHome.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import {Linking, Text, View} from 'react-native';
+import Button from 'components/Button';
+
+import styles from './styles';
+
+type Props = {};
+
+function BlaireHome(props: Props) {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>{`B L A I R E`}</Text>
+      <Button
+        onPress={() => Linking.openURL('https://dintaifungusa.com/')}
```

Comment:
BEST LINK EVER

However, for callbacks please utilize the `useCallback` hook such as the one you've done in `Home.tsx`, i will enable a lint rule for no arrow funcs down the road so the linter can warn you about this

Happy to explain further why this is best practice